<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogChangeShotcuts</class>
 <widget class="QDialog" name="dialogChangeShotcuts">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Клавиатурные сочетания</string>
  </property>
  <property name="locale">
   <locale language="Russian" country="Russia"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_accept">
       <property name="text">
        <string>Принять</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_discard">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Новый файл</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Открыть...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Открыть только для чтения...</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Сохранить...</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Сохранить как...</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Справка</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Выход</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QKeySequenceEdit" name="keySequenceEdit_new">
       <property name="locale">
        <locale language="English" country="UnitedStates"/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QKeySequenceEdit" name="keySequenceEdit_open"/>
     </item>
     <item row="6" column="1">
      <widget class="QKeySequenceEdit" name="keySequenceEdit_exit"/>
     </item>
     <item row="5" column="1">
      <widget class="QKeySequenceEdit" name="keySequenceEdit_help"/>
     </item>
     <item row="4" column="1">
      <widget class="QKeySequenceEdit" name="keySequenceEdit_saveAs"/>
     </item>
     <item row="3" column="1">
      <widget class="QKeySequenceEdit" name="keySequenceEdit_save"/>
     </item>
     <item row="2" column="1">
      <widget class="QKeySequenceEdit" name="keySequenceEdit_openRO">
       <property name="accessibleName">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
